packages/documentation/src/pages/forms/uischema.mdx

```diff
@@ -0,0 +1,448 @@
+
+# uiSchema
```

Comment:
I think a lot of this is copied directly from https://react-jsonschema-form.readthedocs.io/en/latest/api-reference/uiSchema/ and there's a chance not all of it is applicable to our forked version. This should be looked at more closely to ensure only information relevant to our library version is included.